site/layouts/courses/section.html

```diff
@@ -1,6 +1,6 @@
 {{ define "main" }}
 {{ $course_home := printf "%s/%s/%s" "courses" .CurrentSection.Params.course_id "_index.md" }}
-{{ if eq .Dir $course_home }}
```

Comment:
Ah yea, I remember seeing warnings about using .Dir anyway

Reply:
hugo seems to deprecate things a lot... :woman_shrugging: